custom_model_runner/datarobot_drum/drum/root_predictors/predict_mixin.py

```diff
@@ -368,6 +375,25 @@ def do_predict_unstructured(self, logger=None):
         return response, response_status
 
     def do_chat(self, logger=None):
+        unsupported_chat_message = (
+            "This model's chat interface was called, but chat is not supported."
+        )
+        undefined_chat_message = (
+            "This model's chat interface was called, but chat() is not implemented."
+        )
+        # _predictor is a BaseLanguagePredictor attribute of PredictionServer;
+        # see PredictionServer.__init__()
+        if not self._predictor.supports_chat():
+            if self._target_type == TargetType.TEXT_GENERATION:
```

Comment:
I see this has already 2 different messages. Which already makes DR side a bit problematic. It might be that DR side will have only 1 version of this message at least for 11.0

I also don't fully understand the difference between the two.

Reply:
Since we just pass through right now and we have communicated this to Docs team, I guess its fine, but long term needs a solution. @yakov-g  used to suggest error codes